test/electron-locator.ts

```diff
@@ -9,7 +9,7 @@ import { locateElectronModule } from '../src/electron-locator';
 const testElectronRange = require(path.resolve(__dirname, '..', 'package.json')).devDependencies.electron;
 
 function packageCommand(command: string, packageName: string): Promise<string> {
-  return spawn('npm', [command, '--no-save', packageName], {
+  return spawn('npm', [command, '--no-save', '--no-package-lock', packageName], {
```

Comment:
I reverted the yarn change here because it seems that yarn doesn't have a concept of "don't actually update the package.json for this".

Reply:
Update: I completely removed the package manager code in favor of fixtures.